httpony/tests/test_server.py

```diff
@@ -9,5 +9,5 @@ class TestServer(unittest.TestCase):
 
     @mock.patch('httpony.server.run_simple')
     def test_runs(self, run_simple):
-        server.main()
-        self.assertTrue(run_simple.called)
+        server.main(['httpony'])
+        self.assertTrue(run_simple.called)
```

Comment:
Same here. I think the file should have its final line ending. Not sure if your editor did that somehow.